### PR TITLE
Weekly Changelog 2023-09-11 to 2023-09-18

### PR DESCRIPTION
**Pull request by kevingranade**
#### Summary
None

#### Additional context
`Automatically generated as a draft. Please copy-edit before merging.`

### CODE REVIEW

**Comment by Maleclypse** [comment]
Tried a new method of reviewing.  Unsatisfied with the results of adding new commit with my changes already done.  @dseguin  feel free to undo any changes I made in this.  I was trying to figure out how you're changes look so clean compared to mine in the changelogs.  Probably should have just asked.

**Comment by dseguin** [comment]
@Maleclypse Sorry for the late reply. All I do for these changelogs is select multiple lines per category when suggesting changes:

Screenshot_2023-09-23_14-36-06

File: data/changelog.txt [hunk 1]
```diff
@@ -107,6 +108,7 @@ Change description of highlighted regions when editing the overmap
 Detailed information for stats on character creation menu
 Larger and more I18n-friendly safemode UI
 Enable/disable showing several non-player-related messages in the log
+8 new UI colour themes
```
Comment (dseguin) [inline data/changelog.txt:111]:
```suggestion
```

File: data/changelog.txt [hunk 2]
```diff
@@ -145,13 +147,14 @@ Add the Mind Over Matter mod to the CDDA repository
...
 [MoM] Add Enervation damage type, apply it to Eater and feral vitakinetics
-[Magiclysm] Add more than a dozen spells to magiclysm
+[Magiclysm] Add more than two dozen spells to magiclysm
 [Magiclysm] Add fantasy species starting option
 [Magiclysm] Add two more playable fantasy species for Magiclysm
 JSON-ize faction camp hunting returns
 [DinoMod] document lore
 [Aftershock] Rebalance energy weapons to use overheat mechanics
 [Magiclysm] Add fantasy species ferals
+Add dispel magic spells
```
Comment (dseguin) [inline data/changelog.txt:157]:
```suggestion
[Magiclysm] Add dispel magic spells
```

File: data/changelog.txt [hunk 3]
```diff
@@ -252,6 +256,10 @@ Make sure overconfident officers reliably drop their guns
 Fix appliance power drain display right after plugging in/unplugging device
 Fix items and furniture being deleted from grappling hook usage
 Randomly generated characters are now aged appropriately to their profession
+AIM: Display correct truncation of container names
+Fix visible tiles revealing invisible tiles below
+Fix furniture & vehicle map memory refresh
+Fix overflow when using cash card charges
```
Comment (dseguin) [inline data/changelog.txt:262]:
```suggestion
AIM: Display correct truncation of container names
Fix visible tiles revealing invisible tiles below
Fix furniture & vehicle map memory refresh
```

Comment (Maleclypse) [reply data/changelog.txt:262]: Well that's brilliant.  Thanks!

**Pull request merged**